cabinets/parser/txt.py

```diff
@@ -0,0 +1,13 @@
+from cabinets.parser import register_extensions, Parser
+
+
+@register_extensions('txt')
+class PlainTextParser(Parser):
+
+    @classmethod
+    def load_content(cls, content):
```

Comment:
If `dump_content` has an `encoding` kwarg, should `load_content` not have one also? `bytes.decode()` does take an `encoding` keyword arg as well.

Reply:
I was expecting this lol.